pyramid_oereb/core/renderer/getegrid/templates/xml/getegrid.xml

```diff
@@ -12,7 +14,9 @@
     <extract:egrid>${item['egrid']}</extract:egrid>
     <extract:number>${item['number']}</extract:number>
     <extract:identDN>${item['identDN']}</extract:identDN>
-    <extract:type>${item['type']}</extract:type>
+    <data:type>
+        <%include file="real_estate_type.xml" args="real_estate_type=Config.get_real_estate_type_by_data_code(item['type'])"/>
+    </data:type>
```

Comment:
there is some way-but I think this is the case for all our XML output and I tend to not beautify this. This can be adressed in another PR. Here we want to make the content valid again